src/apps/chifra/internal/blocks/handle_count.go

```diff
@@ -21,6 +21,12 @@ func (opts *BlocksOptions) HandleCounts() error {
 	rpcOptions.TransactionWriteDisabled = !opts.CacheTxs
```

Comment:
I really don't like how the management of the cache has "bled up" into the Handler routines. Could we use a `NewStore` function (or `NewRpcOptions`, maybe) that takes parameters such as `chain`, `writeTxs`, and `writeTraces`? That way all this detail could be hidden from the caller. Maybe you already thought of that. Sorry if I'm rushing you...

Reply:
Good catch. I was rushing and forget to refine this part of code